s30899-PatrykSzachniewicz/Kolokwium powtorzenie/zadania

```diff
@@ -0,0 +1,115 @@
+0.Przypuscmy, ze nie mozesz utworzyc zadnego pliku na swoim koncie uczelnianym np. na szuflandii.
+Co sie moglo stac? (wymien co najmniej 3 opcje)
+
+Zakladamy, ze mozesz zalogowac sie w konsoli i nie masz zaleglosci w oplatach.
+
+Powody, przez ktore nie bedziemy mogli utworzyc zadnego pliku na swoim koncie uczelnianym np.
+na szuflandii:
+* brak faktycznego miejsca na dysku
+* brak uprawnien
+* uszkodzenie dysku
+* wykorzystanie calego quota
+
+1.Jest dane takie drzewo katalogow:
+
+.
+├── ala
+│    ├── i
+│    │   └── as
+│    └── ma
+│        └── kota
+└── kot
+     └── ma
+         └── ale
+
+Znajdujesz sie w katalogu kota. Katalog ala, jest w katalogu /tmp/ukos.
+Jak przejsc do katalogu ala ale za pomoca:
+
+-sciezki wzglednej (relatywnej)
+-sciezki bezwzglednej (absolutnej)
+
+Przejscie do katalogu 'ala' za pomoca sciezki wzglednej wykonamy przy pomocy komendy:
+* cd ../../
+natomiast za pomoca sciezki bezwglednej, uzywajac komendy:
+* cd /tmp/ukos/ala
+
+2.Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak utworzyc poddrzewo 
+katalogow jan/kowalski w katalogu ale za pomoca jednej komendy?
+
+Podaj rozwiazania uzywajac sciezek absolutnych i relatywnych.
+
+Aby utworzyc taki uklad katalogow za pomoca sciezki bezwglednej skorzystamy z komendy:
+* mkdir -p /tmp/ukos/kot/ma/ale/jan/kowalski
+natomiast za pomoca sciezki wzglednej komenda bedzie nastepujaca:
+* mkdir -p ../../../kot/ma/ale/jan/kowalski
+
+3.Masz taki uklad katalogow, jak wyzej i ciagle jestes w katalogu kota. Jak przeniesc katalog ale do katalogu i uzywajac:
+-zrodlo (sciezka wzgledna), miejsce docelowe (sciezka absolutna)
+-zrodlo (sciezka absolutna), miejsce docelowe (sciezka wzgledna)
+
+W pierwszym przypadku uzyjemy komendy:
+* mv ../../../kot/ma/ale /tmp/ukos/ala/i
+natomiast w drugim komendy:
+* mv /tmp/ukos/kot/ma/ale ../../i
+
+4.Jak zamknac program, ktory nie reaguje na ctrl+c ?
+
+W przypadku kiedy program nie reaguje na ctrl+c a chcemy go zamknąć należy:
+* uzyc polecenia kill -9, aby wyslac sygnal kill do zamkniecia programu
+* wcisnac ctrl+z w celu wstrzymania programu i przeniesienia go do tla, aby nie obciazal procesora
+
+5.Jak wypisac wszystkie pliki w biezacym katalogu, ktorych nazwa zaczyna sie od al?
+
+Mozna zrobic to uzywajac na przyklad:
+* ls al*
```

Comment:
To za mało. `ls al*` zwróci całą zawartość pasującą do wzorca w tym i katalogi.
Polecam rozwiązania poszukać w `man ls` oraz skorzystać późniejszego filtrowania wirszy.